README.md

```diff
@@ -78,7 +80,7 @@ map.on('load', function() {
 
 https://www.mapbox.com/mapbox-gl-js/example/mapbox-gl-draw/
 
-### See [API.md](https://github.com/mapbox/mapbox-gl-draw/blob/main/docs/API.md) for complete reference.
```

Comment:
I think we can still have a heading here to make it stand out

Reply:
Fair enough, I'll put it back